napari/layers/_image_layer/model.py

```diff
@@ -332,9 +331,9 @@ def _update_thumbnail(self):
             )
             if image.shape[2] == 4:  # image is RGBA
                 downsampled[..., 3] = downsampled[..., 3] * self.opacity
-                colormapped = img_as_ubyte(downsampled)
+                colormapped = downsampled
```

Comment:
if these are now equivalent, should we refactor too?

Reply:
Good call. I took a crack at it Let me know if this is what you had in mind!